### DISCUSSION TITLE
sh: supabase: not found

### DISCUSSION THREAD

**Discussion by aimarsouris** [supabase/supabase#13055] [Questions]
hi,
i npm installed supabase on my WSL Alpine but "supabase init" don't work, i get this error:
/bin/ash: supabase: not found

Did anyone had the same/similar issue
Thanks

**Comment by sweatybridge** [comment]
Does `npx supabase init` work?

**Reply by irfankurniawansuthiono** [reply]
OMG it WORKED lol i just forget npx does exist
